### PR TITLE
205 bug comments render beyond footer on cv

### PR DESCRIPTION
**Pull request by Aradon267**
See issue comments for the help thing

### CODE REVIEW

File: cv_next/app/cv/[cvId]/components/commentSection/comment.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ import { setResolved } from "@/app/actions/comments/setResolved";
 import { deleteComment } from "@/app/actions/comments/deleteComment";
 import Alert from "../../../../../components/ui/alert";
 import Tooltip from "../../../../../components/ui/tooltip";
+//what the ^
```
Comment (Aradon267) [inline cv_next/app/cv/[cvId]/components/commentSection/comment.tsx:21]: Fix these

Comment (YechiamW3) [reply cv_next/app/cv/[cvId]/components/commentSection/comment.tsx:21]: If the `what the` refers to the relative import hell, you can just fix it with `@`s.

File: cv_next/app/cv/[cvId]/components/cvData.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default function CvData({ cv }: { cv: CvModel }) {
     ? uploader.username
     : uploader.full_name;
   return (
-    <div className="grid grid-cols-1 gap-y-4 md:grid-cols-[65%_35%] md:gap-x-4">
+    <div className="grid grid-cols-1 gap-y-4 md:grid-cols-[70%_30%] md:gap-x-4">
```
Comment (YechiamW3) [inline cv_next/app/cv/[cvId]/components/cvData.tsx:14]: Confirm for me that it looks good on mobile as well?
Relevant to all changes in the PR.

File: cv_next/app/hall/page.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ const hall: React.FC = () => {
   return (
     <div className="relative mx-auto h-full w-[700px] max-w-full xl:w-[1400px]">
       <ScrollToTop />
-      <div style={{ textAlign: "center", padding: "20px" }}>
+      <div style={{ textAlign: "center", padding: "5px" }}>
```
Comment (YechiamW3) [inline cv_next/app/hall/page.tsx:8]: Please replace this with tw classes, let's be consistent.

File: cv_next/components/ui/cvPreview.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ export const CvPreview = ({ cv }: { cv: CvModel }) => {
   const previewUrl = transformToPreviewLink(cv.document_link);
 
   return (
-    <div style={{ overflow: "scroll" }}>
+    <div style={{}}>
```
Comment (YechiamW3) [inline cv_next/components/ui/cvPreview.tsx:7]: Are you sure you want to remove this overflow property?
And if you are, you can delete the entire `style` property..

Comment (Omria09) [reply cv_next/components/ui/cvPreview.tsx:7]: As far as I'm concerned there is a built-in scroll support for i-frame pdf reader in all browsers.
removed the style property

Comment (YechiamW3) [reply cv_next/components/ui/cvPreview.tsx:7]: Woops, I completely missed the point when we switched the preview from regular Image to iframe.

File: cv_next/app/cv/[cvId]/components/commentSection/comment.tsx (hunk 1 above)
Comment (Omria09) [reply cv_next/app/cv/[cvId]/components/commentSection/comment.tsx]: done

File: cv_next/app/cv/[cvId]/components/cvData.tsx (hunk 2 above)
Comment (Omria09) [reply cv_next/app/cv/[cvId]/components/cvData.tsx:14]: image

**Review by YechiamW3 — APPROVED** [review]

**Pull request merged**